RestrictEvents/RestrictEvents.cpp

```diff
@@ -115,6 +116,13 @@ struct RestrictEventsPolicy {
 		if (vn_getpath(vp, path, &pathlen) == 0) {
 			//DBGLOG("rev", "csValidatePage %s", path);
 
+			if (modelFindPatch != nullptr && getKernelVersion() >= KernelVersion::Ventura && UNLIKELY(strcmp(path, binPathAboutExtension) == 0)) {
```

Comment:
I am not sure if `binPathSystemInformation` below still needs to be patched on 13.0+, as the error message is now in the new `AboutExtension`. Though, it is of no harm to apply the patch anyway.

Reply:
Good morning. If ii’s useful, System Information works fine, memory info is ok, trouble exists only in the new AboutExtension.

Reply:
UPDATE - Looks like System Information is still used. Resolved.